src/root/index.tsx

```diff
@@ -16,6 +16,12 @@ import Routes from './Routes';
 import 'locales/i18nextConfig';
 import 'styles/index.scss';
 
+import moment from 'moment';
+
+moment.defaultFormat = 'YYYY.MM.DD';
+
+window['moment'] = moment;
```

Comment:
when we start implementing SSR, there is no window object on the server. It will cause app break in case of trying to assign value to it